packages/cli/test/mocks/deployment.ts

```diff
@@ -12,12 +12,32 @@ export function useDeployment({
 }: {
   creator: Pick<User, 'id' | 'email' | 'name'>;
 }) {
+  type state =
```

Comment:
This should be capital `State` since its a type. Also you can probably use the existing type instead

```ts
type State = Deployment['readyState'];
```